app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,217 @@
+@import url('https://fonts.cdnfonts.com/css/proxima-nova-condensed');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f3f2f2;
```

Comment:
- [optional] Great job. Please kindly adjust the category selection section when trying to create a new transaction. It is overflowing . Remember, you are replicating someone's design and following all sections will look professional. 
![Screenshot_25](https://github.com/danielsafari143/budget-app/assets/65298048/d13e2e68-5d5e-441a-a56c-d46e4a6b07ae)

Reply:
Thank you sir for this feedback .